spec/models/price_group_spec.rb

```diff
@@ -71,7 +71,7 @@
       it "should be deletable if no orders on policy" do
         expect(@price_group).to be_can_delete
         @price_group.destroy
-        expect(@price_group).to be_destroyed
+        expect { PriceGroup.find(@price_group.id) }.to raise_error(ActiveRecord::RecordNotFound)
```

Comment:
I think it might be a good idea to add a test that verifies that it is still there.

`expect(PriceGroup.with_deleted.find(@price_group.id)).to be_present`